gateway/engine/core/src/main/java/io/apiman/gateway/engine/impl/DefaultPolicyErrorWriter.java

```diff
@@ -67,7 +65,7 @@ public void write(ApiRequest request, Throwable error, IApiClientResponse respon
             isXml = true;
         }
         String message = createErrorMessage(request, error);
-        logger.error(message, error);
+        logger.error("DefaultPolicyErrorWriter: "+ message, error);
```

Comment:
Did you mean to include this? If so, should it not be handled by the logger formatter's configuration (e.g. log4j config). I just notice we use the name of the class already when constructing the logger.

Reply:
No, I think I just wanted to keep the other one, this one was just for debugging. But I made these changes before the Christmas holidays and just contributed them today :)